### ISSUE TITLE
Make documentation more accessible

### ISSUE THREAD

**Issue by salkinium** [modm-io/lbuild#52]
Currently the Readme contains the whole documentation, which is fairly complete, but not super useful to access.

Wishlist:

- Have a proper website at lbuild.modm.io or readthedocs with overview, install instructions, development instructions etc.
- Split the current Readme documentation into smaller and *searchable* modules.
- Find a way to install the docs with `pip install lbuild`
- Manpages? or at least a better `lbuild -h`.

**Comment by rleh** [comment]
> Find a way to install the docs with pip install lbuild

Once the documentation is available on https://lbuild.modm.io/ I don't think we need the documentation in a pip package. `lbuild -h` could then refer to the website.

**Comment by rleh** [comment]
For a start: What do you think about showing the current [README](https://github.com/modm-io/lbuild/blob/develop/README.md) on https://lbuild.modm.io/ and pointing to a (non-empty) ReadTheDocs there?

**Comment by salkinium** [comment]
Yes, just currently working on too many things at once, hence opening this issue.
